src/ffishjs.cpp

```diff
@@ -41,11 +41,12 @@
 using namespace emscripten;
 
 
-void initialize_stockfish(std::string& uciVariant) {
+void initialize_stockfish() {
   pieceMap.init();
   variants.init();
   UCI::init(Options);
-  PSQT::init(variants.find(uciVariant)->second);
+  for (std::string uciVariant : variants.get_keys())
+    PSQT::init(variants.find(uciVariant)->second);
```

Comment:
The initialization of PSQT always overwrites the current values, i.e., only one variant can be active at a time. Therefore the for loop does not make much sense here, but since PSQT anyway are not relevant for ffish.js, it should not have any impact either.

Reply:
Makes sense. I removed the PSQT initialisation